app/customer.py

```diff
@@ -0,0 +1,79 @@
+from app.shop import GetShopsData
+from app.car import GetCarsData
+
+
+class GetCustomersData:

```

Comment:
Same here. Your class should be equal to real object. Like Customer
GetCustomersData is not an some object. Fix that. OOP